File: conditionals/time_of_day.py
# Malu Estevam, Time of Day
import time

current = time.time()
local_time = time.localtime(current)
hour = local_time.tm_hour


if hour > 6 or hour <= 11:
    print("Good morning!")
elif hour > 11 or hour <= 4:
    print("Good afternoon!")
elif hour > 4 or hour <= 7:
    print("Good evening!")
else:
    print("Go to bed, it's late")

File: conditionals/time_of_day.c
//Malu Estevam, Template
#include <stdio.h>
#include <time.h>




int main(void){
    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;

    if(hour > 6 || hour <= 11){
        printf("Good morning!");
    }
    else if(hour > 11 || hour <= 16){
        printf("Good afternoon!");
    }
    else if(hour > 16 || hour<= 19){
        printf("Good evening!");
    }else{
        printf("Go to bed. It is late");
    }
    return 0;
}